### PR TITLE
Feat/ticket form

### PR DESCRIPTION
**Pull request by ngqtrieu99**

### CODE REVIEW

File: bookingticketui/src/components/ticket-form/ticket-form.jsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            {show ? (
+              <div
+                style={{
+                  display: "flex",
+                  gap: "10px",
+                  flexWrap: "wrap",
+                  justifyContent: "center",
+                }}
+              >
+                {tickets.map((getcon) => (
```
Comment (tanngocIT) [inline bookingticketui/src/components/ticket-form/ticket-form.jsx:98]: getcon??

Comment (ngqtrieu99) [reply bookingticketui/src/components/ticket-form/ticket-form.jsx]: fixed. Changed the name to "items" to make more sense

File: bookingticketui/src/components/ticket-form/ticket-form.jsx [hunk 2]
```diff
@@ -84,60 +87,73 @@ const Ticket = () => {
...
+                            new Date(
+                              data.result?.data?.tickets[0].departureTime
+                            ),
+                            "dd/MMMM/yyyy"
+                          )}
+                        </ListGroup.Item>
+
+                        <ListGroup.Item>
+                          Seat Type: {data.result?.data?.tickets[0].seatClass}
+                        </ListGroup.Item>
```
Comment (hnthinh55) [inline bookingticketui/src/components/ticket-form/ticket-form.jsx:137]: ticket[index] is not necessary here when you already have "map". Take advantage of it

File: bookingticketui/src/components/ticket-form/ticket-form.jsx [hunk 3]
```diff
@@ -84,60 +87,73 @@ const Ticket = () => {
...
+
+                        <ListGroup.Item>
+                          Depart location:{" "}
+                          {data.result?.data?.tickets[0].locationFrom}
```
Comment (hnthinh55) [inline bookingticketui/src/components/ticket-form/ticket-form.jsx]: here too

File: bookingticketui/src/components/ticket-form/ticket-form.jsx [hunk 4]
```diff
@@ -50,6 +51,8 @@ const Ticket = () => {
       type = "success";
       displayAlert(message, type);
       setData(result);
```
Comment (hnthinh55) [inline bookingticketui/src/components/ticket-form/ticket-form.jsx]: should change to set(result.result?.data?)

**Review by hnthinh55 — APPROVED** [review]

**Pull request merged**